lib/pr.js

```diff
@@ -4,6 +4,25 @@ const core = require('@actions/core');
 
 const MARKER = "=== DO NOT EDIT BELOW THIS LINE ===";
 
+const emptyLineRegex = /^\s*$/;
+
+function trimEmptyLines(body) {
+  const result = [];
+  let isPreviousEmpty = true;
```

Comment:
I put `true` so we will not have a body starting with a blank line. About the logic, it will collapse all blank line to a single one (same as github preview does)

Reply:
About the changelog body it's updated here https://github.com/nugit/actions-pr-changelog/pull/9/files

Reply:
Okay, my bad. I read the logic wrongly. 😓